modin/pandas/test/test_io.py

```diff
@@ -860,6 +861,135 @@ def test_read_csv_iteration(self, make_csv_file, iterator):
 
         df_equals(modin_df, pd_df)
 
+    # Quoting, Compression, and File Format parameters tests
+    @pytest.mark.parametrize("compression", ["infer", "gzip", "bz2", "xz", "zip"])
+    @pytest.mark.parametrize(
+        "encoding",
+        [None, "latin8", "ISO-8859-1", "latin1", "iso-8859-1", "cp1252", "utf8"],
+    )
+    @pytest.mark.parametrize("engine", [None, "python", "c"])
+    def test_read_csv_compression(self, make_csv_file, compression, encoding, engine):
```

Comment:
Why are encoding and compression tests combined?

Reply:
@anmyachev The combination of these two parameters is important to test so we ensure that when the file is compressed in a non UTF-8 format it can be properly read by Modin.

Reply:
got it, thanks @devin-petersohn